src/i18n/locales/cs/translation.json

```diff
@@ -741,7 +741,7 @@
     "success": "Úspěch",
     "error": "Chyba",
     "settings": "Nastavení",
-    "websites": "Webové stránky",
+    "connected_sites": "Connected Sites",
```

Comment:
Rename the key, but leave the translation empty. 

```suggestion
    "connected_sites": "",
```

Reply:
Apply this for all translations, please.

Reply:
ok sure